go.mod

```diff
@@ -71,6 +71,8 @@ require (
 	software.sslmate.com/src/go-pkcs12 v0.2.0
 )
 
+require github.com/nrdcg/nodion v0.1.0
```

Comment:
This looks out of place. Doesn't `go mod tidy` separate the dependencies in two groups (direct and indirect dependencies) anymore?